content/_data/events/2019-08-12-devops-world-jenkins-world-sf.adoc

```diff
@@ -0,0 +1,6 @@
+---
+title: "DevOps World | Jenkins World 2019 San Francisco"
```

Comment:
I am not sure whether it fits the Event title length limitation

Reply:
Thanks Oleg I will fix the other as well hopefully this works better